graphsdk/src/main/java/com/microsoft/graph/model/Date.java

```diff
@@ -0,0 +1,90 @@
+package com.microsoft.graph.model;
+
+
+import java.util.Locale;
+
+/**
+ * A timezone-nonspecific date
+ */
+public class Date {
```

Comment:
Add a ctor that takes (int year, int month, int day) so its easy for devs to new up there own value